src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -2190,6 +2190,38 @@ private function addMailerSection(ArrayNodeDefinition $rootNode, callable $enabl
                             ->end()
                         ->end()
                     ->end()
+                    ->children()
+                        ->arrayNode('signer')
+                            ->info('Signer configuration')
+                            ->children()
+                                ->arrayNode('dkim')
+                                    ->info('DKIM configuration')
+                                    ->children()
+                                        ->scalarNode('key')->defaultNull()->end()
+                                        ->scalarNode('domain')->defaultNull()->end()
+                                        ->scalarNode('select')->defaultNull()->end()
+                                        ->scalarNode('passphrase')->defaultNull()->end()
+                                        ->arrayNode('options')
+                                            ->normalizeKeys(false)
```

Comment:
this probably need `->useAttributeAsKey('name')`, as I would expect options to be an associative array.